lib/xandra/protocol.ex

```diff
@@ -714,7 +721,12 @@ defmodule Xandra.Protocol do
     {{:set, [type]}, buffer}
   end
 
-  # TODO: UDT
+  defp decode_type(<<0x0030::16>> <> buffer) do
+    {_keyspace, buffer} = decode_string(buffer)
+    {_name, <<count::16>> <> buffer} = decode_string(buffer)
```

Comment:
This works fine, but I would prefer to have this parsing split in two steps, one where we parse the name of the UDT and one where we parse `n`:

```elixir
{_name, buffer} = decode_string(buffer)
<<count::16>> <> buffer = buffer
```

I think it makes the code a bit clearer without hiding the "double parsing" on one line IMO. Wdyt?